### PR TITLE
Added support for terminals without Emojis.

### PR DESCRIPTION
**Pull request by sjs7007**
Trying to fix the headers not being displayed [issue](https://github.com/jaebradley/uber-cli/issues/22). It determines whether emojis are supported or not by checking OS type. Only Mac OS is assumed to support emojis but we can easily add other OS'es that might support emojis.

### CODE REVIEW

File: src/services/tables/builders/PriceEstimatesTableBuilder.js [hunk 1]
```diff
@@ -6,8 +6,33 @@ import Table from 'cli-table2';
 
 import Utilities from '../../../Utilities';
 
+var emojiCarType = emoji.get('oncoming_automobile');
```
Comment (jaebradley) [inline src/services/tables/builders/PriceEstimatesTableBuilder.js]: @sjs7007 thank you for doing this (and sorry about the wait)!

I wonder, and maybe this is total overkill, but do you think some sort of "symbol" service might make this logic a little cleaner? This service would abstract away the `process.platform` call.

Let me know your thoughts! If you're busy, I'm more than happy to do a first pass on what this service might look like.

**Comment by sjs7007** [comment]
Hey. Thanks for replying. I'm indeed a bit busy these days so maybe you can have a go at it earlier instead. Thanks!

**Comment by jaebradley** [comment]
@sjs7007 so I've implemented the beginnings of a symbol service on the [`symbol-service` branch](https://github.com/jaebradley/uber-cli/tree/symbol-service). 

I want to still give you credit for your contributions on this PR though.

I **think** in order to do that you need to
* Update your fork (currently, there are merge conflicts - basically, I've renamed those two service classes)
* I can PR my `symbol-service` branch into your fork's
* You open a PR from your fork's `master` against my `master`

Let me know if I'm totally off here - this is my first time dealing with a PR from a fork that's out-of-date that I want to make changes against before merging.

How does that sound? If you're too busy, I can just get these changes merged in, but I wanted to give you the option.

**Comment by jaebradley** [comment]
@sjs7007 bumping this thread / PR - any additional thoughts? I want to get these changes in sooner rather than later, but I also want to give you credit for your contributions as well.

**Comment by jaebradley** [comment]
@sjs7007 I have #53 if you'd like to use that in your forked `master`. If not, I'm inclined to merge #53 in sooner rather than later and close this PR. Would really like for you to update your `master` so you can get credit for your contributions.

**Comment by sjs7007** [comment]
Sorry @jaebradley, I just moved to a new city and started a new job some time back. Quite busy and didn't get time to deal with this. Thanks for giving me so many options though!

**Comment by jaebradley** [comment]
Oh man, @sjs7007 I'm sorry I just closed this and merged in my changes. Good luck moving to the new city and job - let me know if you want to work on something else.

**Pull request closed without merging**